server/helpers/graphql/queries/delegate.js

```diff
@@ -4,7 +4,7 @@ const gql = require('graphql-tag')
 const _ = require('lodash')
 
 // Returns the delegate summary, does not include rewards, ROI, missed reward calls or any calculated data
-const getDelegateSummary = async delegateAddress => {
+const getLivepeerTranscoderAccount = async delegateAddress => {
```

Comment:
Req: Maybe is a good idea to use `getLivepeerDelegateAccount`  for this function

Reply:
Good idea, it's done now, I'd also changed the name of the `getLivepeerTranscoders`